src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParserStateObjectNative.cs

```diff
@@ -425,12 +425,16 @@ internal override uint WaitForSSLHandShakeToComplete(out int protocolVersion)
             var nativeProtocol = (NativeProtocols)nativeProtocolVersion;
 
             /* The SslProtocols.Tls13 is supported by netcoreapp3.1 and later
-             * This driver does not support this version yet!
+             * This driver does not support this version yet! */
+#if NETCOREAPP3_1_OR_GREATER
```

Comment:
Since we are not supporting netcoreapp3.1 it should suffice if we just want to check for NETCOREAPP and omit the version.
```suggestion
#if NETCOREAPP
```